modules/elements-color-picker/assets/js/editor/commands/start.js

```diff
@@ -0,0 +1,21 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class Start extends CommandBase {
+	apply( args ) {
+		elementor.$previewContents.find( 'body' ).addClass( 'elementor-editor__ui-state__color-picker' );
+
+		this.component.currentPicker = {
+			container: args.container,
+			control: args.control,
+			initialColor: args.container.getSetting( args.control ),
```

Comment:
Use `args.container.settings.get(  args.control )` in order to get only the local setting. (in case it's a global color)

Reply:
`getSetting()` gives me the initial color also for global color.